ReactCommon/cxxreact/Instance.h

```diff
@@ -106,14 +107,10 @@ class RN_EXPORT Instance {
   void callNativeModules(folly::dynamic &&calls, bool isEndOfBatch);
   virtual void loadApplication(std::unique_ptr<RAMBundleRegistry> bundleRegistry,
                        std::unique_ptr<const JSBigString> bundle,
-                       uint64_t bundleVersion, // TODO(OSS Candidate ISS#2710739)
-                       std::string bundleURL,
-                       std::string&& bytecodeFileName); // TODO(OSS Candidate ISS#2710739)
+                       std::string bundleURL);
   virtual void loadApplicationSync(std::unique_ptr<RAMBundleRegistry> bundleRegistry,
                            std::unique_ptr<const JSBigString> bundle,
-                           uint64_t bundleVersion, // TODO(OSS Candidate ISS#2710739)
-                           std::string bundleURL,
-                           std::string&& bytecodeFileName); // TODO(OSS Candidate ISS#2710739)
+                           std::string bundleURL);
```

Comment:
>g [](start = 37, length = 1)

&&?

Reply:
Not wanting to change existing interfaces for upstream RN. See other comment.

---
In reply to: [342831298](https://github.com/microsoft/react-native/pull/187#discussion_r342831298) [](ancestors = 342831298)